viewers/nerf_viewer/bridge_server/bridge_server_subprocess.py

```diff
@@ -0,0 +1,85 @@
+import atexit
```

Comment:
Shall we rename it to `bridge_server.py`? As there is no conflict, it is not necessary to add `_subprocess` suffix.

Reply:
1. Renamed `bridge_server_subprocess.py` to `bridge_server.py`
2. Renamed function `run_bridge_server_as_subprocess` to `start_bridge_server`